### ISSUE TITLE
Désactiver automatiquement les bêtas périmées à la publication

### ISSUE THREAD

**Issue by SpaceFox** [zestedesavoir/zds-site#4269]
Quand on a un contenu en bêta, et qu'on publie cette version (ou une version postérieure) du contenu, on se retrouve avec une bêta obsolète et rien qui ne signifie clairement qu'il existe encore une version en bêta du contenu publié.

Il faudrait donc que :

- Si la version en bêta est la version publiée ou une version plus ancienne, la bêta soit automatiquement désactivée.
- Si la version en bêta est postérieure à la version publiée (cas d'un big-tuto qu'on continue à écrire et qui a des publications partielles, par exemple) la bêta soit laissée active.

**Comment by artragis** [comment]
pour l'instant les béta de tous les articles sont désactivées à la publication.

**Comment by pierre-24** [comment]
Ouais, mais le comportement que propose Spacefox est pas déconnant :)

**Comment by artragis** [comment]
Yep, c'était juste pour info.

**Comment by SpaceFox** [comment]
On a donc un comportement différent entre article et tuto ?!

**Comment by pierre-24** [comment]
Tout à fait. Voulu et assumé, justement sur la base qu'un article avait peu de chance d'évoler, là ou un tuto oui :)

**Comment by Arnaud-D** [comment]
Je pense que ça serait mieux d'avoir le même comportement pour les deux, à savoir fermer la bêta à la publication.

Les tutos ne sont pas forcément modifier fréquemment, ni dans la foulée d'une publication. Il sufit à l'auteur de réactiver la bêta quand il a du travail à présenter en bêta, ce qui en général n'arrive pas immédiatement après la publication.

D'autres avis ?
